Tests/Controller/CaptureControllerTest.php

```diff
@@ -38,7 +38,18 @@ public function shouldBeSubClassOfController()
      */
     public function throwBadRequestIfSessionNotStartedOnDoSessionAction()
     {
-        $controller = new CaptureController;
+        $this->registryMock = $this->createMock(RegistryInterface::class);
+        $this->httpRequestVerifierMock = $this->createMock(
+            HttpRequestVerifierInterface::class
+        );
+        $this->payum = new Payum(
```

Comment:
This should already be handled in the `AbstractControllerTest`, so it's not necessary to re-define the mock again here (Once the `initMocks` is updated to `setUp`)